tests/services/label.service.spec.ts

```diff
@@ -1,9 +1,56 @@
 import { LabelService } from '../../src/app/core/services/label.service';
 import { Label } from '../../src/app/core/models/label.model';
 import * as LabelConstant from '../constants/label.constants';
+import { of } from 'rxjs';
 
 let labelService: LabelService;
 let labelList: Label[];
+let githubService: any;
+
+describe('LabelService', () => {
+    beforeEach(() => {
+      githubService = jasmine.createSpyObj('GithubService', ['fetchAllLabels', 'createLabel']);
+      labelService = new LabelService(githubService);
+    });
+
+    describe('.syncLabels()', () => {
+      it('should create all required labels if no required labels are fetched', () => {
+        githubService.fetchAllLabels.and.callFake(() => of([]));
+        of(true)
+          .pipe(labelService.syncLabels())
+          .subscribe((result: {}[]) => expect(result).toEqual([]));
```

Comment:
```suggestion
          .subscribe();
```
I don't think there is a large benefit in checking this response, especially since we're mocking `fetchAllLabels`. 
Let's remove it?

Reply:
Can!